src/Xunit.Combinatorial/ValuesUtilities.cs

```diff
@@ -21,6 +21,13 @@ internal static IEnumerable<object> GetValuesFor(ParameterInfo parameter)
             Requires.NotNull(parameter, nameof(parameter));
 
             var valuesAttribute = parameter.GetCustomAttribute<CombinatorialValuesAttribute>();
+            var rangeAttribute = parameter.GetCustomAttribute<CombinatorialRangeAttribute>();
+
+            if (rangeAttribute != null)
+            {
+                return rangeAttribute.Values;
+            }
+
```

Comment:
This is good, but you inserted it in between the existing local and its use. Can you move this to *after* the use of `valuesAttribute`?

Reply:
Rearranged the lines, thanks for your help!